web-app/js/portal/details/NcWmsPanel.js

```diff
@@ -324,7 +324,46 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
         var dateRangeStart = this._getDateFromPicker(this.startDateTimePicker);
         var dateRangeEnd = this._getDateFromPicker(this.endDateTimePicker);
 
-        this.dataCollection.updateNcwmsParams(dateRangeStart, dateRangeEnd, geometry);
+        this.dataCollection.setFilters(this._ncwmsParamsAsFilters(dateRangeStart, dateRangeEnd, geometry));
+    },
+
+    _ncwmsParamsAsFilters: function(dateRangeStart, dateRangeEnd, geometry) {
```

Comment:
Is this perhaps worth of it's own class?  In fact, can we introduce a `Portal.filter.NcwmsDateFilter` (or whatever) - then this starts to be more like the other filter panel - and should make it easier to merge the two.  I'm not really up with the filter and filter UI design at the moment though, so it's just a thought.